.circleci/config.yml

```diff
@@ -3,8 +3,7 @@ version: 2.1
 orbs:
   hokusai: artsy/hokusai@volatile
   horizon: artsy/release@volatile
-  node: circleci/node@6.3.0
-  yarn: artsy/yarn@6.5.0
+  node: circleci/node@7.2.1
```

Comment:
This is the way @ovasdi 👍 - much better, all of the artsy/yarn stuff is deprecated and we've removed its use from other apps. CircleCI orb does all the things, caching, etc.